jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -129,84 +128,82 @@ ecma_round_high_to_uint64 (ecma_uint128_t *num_p)
  *
  * Q = Q3 *2^96  + Q2 *2^64  + Q1 *2^32  + Q0 *2^0     // 128-bit quotient
  */
-#define ECMA_UINT128_DIV10(name) \
-{ \
-  /* estimation of reciprocal of 10, 128 bits right of the binary point (T1 == T2) */ \
-  const uint64_t tenth_l = 0x9999999aul; \
-  const uint64_t tenth_m = 0x99999999ul; \
-  const uint64_t tenth_h = 0x19999999ul; \
-  \
-  uint64_t l0 = ((uint32_t) name.lo) * tenth_l; \
-  uint64_t l1 = (name.lo >> 32u) * tenth_l; \
-  uint64_t l2 = ((uint32_t) name.hi) * tenth_l; \
-  uint64_t l3 = (name.hi >> 32u) * tenth_l; \
-  uint64_t m0 = ((uint32_t) name.lo) * tenth_m; \
-  uint64_t m1 = (name.lo >> 32u) * tenth_m; \
-  uint64_t m2 = ((uint32_t) name.hi) * tenth_m; \
-  uint64_t m3 = (name.hi >> 32u) * tenth_m; \
-  uint64_t h0 = ((uint32_t) name.lo) * tenth_h; \
-  uint64_t h1 = (name.lo >> 32u) * tenth_h; \
-  uint64_t h2 = ((uint32_t) name.hi) * tenth_h; \
-  uint64_t h3 = (name.hi >> 32u) * tenth_h; \
-  \
-  uint64_t q0 = l0 >> 32u; \
-  q0 += (uint32_t) l1; \
-  q0 += (uint32_t) m0; \
-  \
-  q0 >>= 32u; \
-  q0 += l1 >> 32u; \
-  q0 += m0 >> 32u; \
-  q0 += (uint32_t) l2; \
-  q0 += (uint32_t) m1; \
-  q0 += (uint32_t) m0; \
-  \
-  q0 >>= 32u; \
-  q0 += l2 >> 32u; \
-  q0 += m1 >> 32u; \
-  q0 += m0 >> 32u; \
-  q0 += (uint32_t) l3; \
-  q0 += (uint32_t) m2; \
-  q0 += (uint32_t) m1; \
-  q0 += (uint32_t) h0; \
-  \
-  q0 >>=32u; \
-  q0 += l3 >> 32u; \
-  q0 += m2 >> 32u; \
-  q0 += m1 >> 32u; \
-  q0 += h0 >> 32u; \
-  q0 += (uint32_t) m3; \
-  q0 += (uint32_t) m2; \
-  q0 += (uint32_t) h1; \
-  \
-  uint64_t q1 = q0 >> 32u; \
-  q1 += m3 >> 32u; \
-  q1 += m2 >> 32u; \
-  q1 += h1 >> 32u; \
-  q1 += (uint32_t) m3; \
-  q1 += (uint32_t) h2; \
-  \
-  uint64_t q32 = q1 >> 32u; \
-  q32 += m3 >> 32u; \
-  q32 += h2 >> 32u; \
-  q32 += h3; \
-  \
-  name.lo = (q1 << 32u) | ((uint32_t) q0); \
-  name.hi = q32; \
-}
-
-#if defined (__GNUC__) || defined (__clang__)
+#define ECMA_UINT128_DIV10(name)                                                        \
+  {                                                                                     \
```

Comment:
Incorrect syntax, the slash should be kept in its place.

Reply:
Slash is aligned no the longest line of the macro, so still correct.